payload-command/cmd/render/main.go

```diff
@@ -11,7 +11,9 @@ import (
 // this command injects the initial FeatureGate.status and places some CRDs to be created by the installer during bootstrapping
 // remember that these manifests are not maintained in a running cluster.
 func main() {
-	o := &render.RenderOpts{}
+	o := &render.RenderOpts{
+		ImageProvidedManifestDir: "/usr/share/bootkube/manifests/manifests",
```

Comment:
Is this a specific magic location that came from somewhere or something you chose?

Reply:
> Is this a specific magic location that came from somewhere or something you chose?

matched kas-o as I recall.